.github/workflows/install-circt/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Install CIRCT
+
+inputs:
+  version:
+    description: 'version to install'
+    required: false
+    default: 'sifive/1/24/0'
+
+runs:
+  using: composite
+  steps:
+    - id: cache-circt
+      uses: actions/cache@v3
+      with:
+        path: circt
+        key: circt-${{ runner.os }}-${{ inputs.version }}
+
+    - shell: bash
+      if: steps.cache-circt.outputs.cache-hit != 'true'
+      run: |
+        mkdir circt
+        # Escape version forward slashes for use in URL
+        VERSION=$(echo ${{ inputs.version }} | sed 's|/|%2F|g')
```

Comment:
For what it's worth, I figured out that GitHub doesn't require you to escape slashes in the version tag when downloading artifacts. According to the HTTP spec, servers are allowed to differentiate between `/` and `%2F`, but in practice, it looks like GitHub treats both the same.

Reply:
Worthwhile to simplify the code where we can! Thanks!